doc/build/html/_sources/index.rst.txt

```diff
@@ -4,23 +4,51 @@
    contain the root `toctree` directive.
 
 Welcome to The Sheriff's documentation!
-=======================================
+***************************************
+
 
 .. toctree::
    :maxdepth: 2
    :caption: Contents:
 
 
+Introduction
+============
+
+This part of the documentation is intended to birefly introduce
+The Sheriff application.
+
+.. toctree::
+   introduction
+
 Application usage
------------------
+=================
 
 Here's all you need to run and test The Sheriff application.
 
 .. toctree::
     usage
 
+Design - Hexagonal Architecture
+===============================
+Missing explanation of why whe choose this architecture
+
+- The Sheriff Application
+
+Missing explanation of architectural decisions in Application layer
+
+- The Sheriff Domain
+
+Missing explanation of architectural decisions in Domain layer
```

Comment:
Here we put our domain classes (Outlaw, Gang, Raid) containing all pertinent business logic and the factory classes.